### PR TITLE
feat(planner): migrate CreateUDF to planner v2

### LINKED ISSUE #5904: Migrate CREATE FUNCTION statement to new planner #5899
#5778

### PR DESCRIPTION
**Pull request by TennyZhuang**
Signed-off-by: TennyZhuang <zty0826@gmail.com>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

migrate CreateUDF to planner v2

## Changelog

- New Feature
- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #5904

### CODE REVIEW

**Review by leiysky — APPROVED** [review]

File: query/src/sql/planner/format/display_plan.rs [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 use common_exception::Result;
+use jwt_simple::reexports::anyhow::Ok;
```
Comment (BohuTANG) [inline query/src/sql/planner/format/display_plan.rs:16]:
```suggestion

```

**Review by andylokandy — APPROVED** [review]

**Comment by Xuanwo** [comment]
@mergify refresh

**Comment by Xuanwo** [comment]
@mergify update

**Comment by Xuanwo** [comment]
> Stateless tests related to the UDF(in the old planner) need to add to the v2 :/ https://github.com/datafuselabs/databend/blob/87f4250119bb69689a6aa060400f70bfd7cc7073/tests/suites/0_stateless/05_ddl/05_0010_ddl_create_udf.sql

Added.

**Pull request merged**